gtdynamics/optimizer/ConnectedComponent.cpp

```diff
@@ -0,0 +1,33 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  ConnectedComponent.h
+ * @brief Connected component implementations.
+ * @author: Yetong Zhang
+ */
+
+#include <gtdynamics/optimizer/AugmentedLagrangianOptimizer.h>
+#include <gtdynamics/optimizer/ConnectedComponent.h>
+#include <gtdynamics/optimizer/PenaltyMethodOptimizer.h>
+#include <gtdynamics/utils/DynamicsSymbol.h>
+#include <gtsam/nonlinear/LevenbergMarquardtOptimizer.h>
+#include <gtsam/nonlinear/LinearContainerFactor.h>
+
+namespace gtsam {
```

Comment:
Shouldn't this be gtdynamics???

Every file in this PR has this issue...

Reply:
That will require lots of re-writing, since lots of class names will need `gtsam::` to be added.

I envision that this will be going into `gtsam` or a separate repo eventually, so I'll add a TODO here instead of doing the heavy rewritting.

Reply:
But this is a PR to gtdynamics right? If this is going into GTSAM, shouldn't this be a PR for GTSAM then?

Reply:
I strongly believe we should update the namespace here in this PR rather than save it for later.